applovin_max/android/src/main/java/com/applovin/applovin_max/AppLovinMAX.java

```diff
@@ -275,6 +277,15 @@ private void initialize(final String pluginVersion, final String sdkKey, final R
             targetingInterestsToSet = null;
         }
 
+        if ( !extraParametersToSet.isEmpty() )
+        {
+            for ( Map.Entry<String, String> entry : extraParametersToSet.entrySet() )
+            {
+                sdk.getSettings().setExtraParameter( entry.getKey(), entry.getValue() );
+            }
+            extraParametersToSet.clear();
+        }
+
```

Comment:
Here, can you implement [setPendingExtraParametersIfNeeded() ](https://github.com/AppLovin/AppLovin-MAX-React-Native/blob/e471e4db559cecd72daabe4f70903c7c77deafa9/android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java#L2273) instead, as we have in React Native?   We want to keep the same behavior across the plugins we provide.